server/sonar-web/src/main/js/apps/coding-rules/rule/profile-activation-view.js

```diff
@@ -81,7 +81,7 @@ export default ModalForm.extend({
         value: $(this).val() || $(this).prop('placeholder') || ''
       };
     }).get();
-    const paramsHash = (params.map(function (param) {
+    const paramsHash = (params.map(param => {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove curly braces and "return" from this arrow function body. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)